controllers/org_info_metric.go

```diff
@@ -0,0 +1,54 @@
+package controllers
+
+import (
+	"context"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	orgv1 "github.com/appuio/control-api/apis/organization/v1"
+)
+
+//+kubebuilder:rbac:groups="rbac.appuio.io",resources=organizations,verbs=get;list;watch
+//+kubebuilder:rbac:groups="organization.appuio.io",resources=organizations,verbs=get;list;watch
+
+var orgInfoMetricDesc = prometheus.NewDesc(
+	"appuio_control_organization_info",
+	"Information about APPUiO Cloud organizations",
+	[]string{"organization", "billing_entity", "sales_order"},
```

Comment:
Very nitpicky but is it sale order or sales order? #182 most has references to `sale order`.

Reply:
Not even odoo is consistent. It's a `sale.order` in the model, but it's referred to as `Sales Order` in the UI. That's why I've been using `sale order` in the code but went against that convention with the outward facing metric label.

Do you have a preference?